pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -482,7 +482,7 @@ func (c *Reconciler) reconcile(ctx context.Context, pr *v1beta1.PipelineRun, get
 
 	for _, rprt := range pipelineRunFacts.State {
 		if !rprt.IsCustomTask() {
```

Comment:
I don't see much discussion of this in TEP-0090 but I imagine we want to support passing matrixed params into custom tasks. Will they still be passed in, and this is just skipped because the run controller is expected to handle validation?

Reply:
that makes sense to me, thanks Jerop!